elpi.install

```diff
@@ -0,0 +1,97 @@
+lib: [
```

Comment:
Checking the file in is wrong because it contains some platform specific stuff (`.cmxs` aren't necessarily available everywhere), but luckily it's harmless as dune will overwrite it anyway when building the package. I suggest you just remove this file and add it to your .gitignore.

Reply:
I tried with `run`, but it insists in running `camlp5o.opt` that is then unable to load the `cmo`s.